README.rst

```diff
@@ -43,7 +43,7 @@ fr        French                4.0.0*       amuratet           Amuratet?
 id        Indonesian            4.0.0*       ridho1991          Mutasim Ridlo
 it        Italian               4.0.0        luk3b8             Luca Rasia
 ja        Japanese              4.0.0        kohenji01          Shoji Ogura
-ko        Korean                4.0.0*       Timothy-Y-H-Lee    Timothy-Y-H-Lee
+ko        Korean                4.0.4        codeigniter-kr     codeigniter-kr
```

Comment:
```suggestion
ko        Korean                4.0.2        codeigniter-kr     codeigniter-kr
```

Reply:
I checked based on the 4.0.4 repository, is it correct that this is marked as 4.0.2?

Reply:
Yes, this was released in this repo as v4.0.2